webhook_server/tests/test_helpers.py

```
@@ -156,6 +156,45 @@ def test_get_api_with_highest_rate_limit_invalid_tokens(
+        mock_rate_limit.rate.reset = Mock()
+        mock_rate_limit.rate.limit = 5000
+        mock_api.get_rate_limit.return_value = mock_rate_limit
+
+        mock_get_apis.return_value = [(mock_api, "token1")]
+
+        with patch.dict(os.environ, {"WEBHOOK_SERVER_DATA_DIR": "webhook_server/tests/manifests"}):
+            config = Config(repository="test-repo")
+            api, token, user = get_api_with_highest_rate_limit(config=config, repository_name="test-repo")
+
+        assert api == mock_api
+        assert token == "token1"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "token"

(S105)